Sprint-3/2-practice-tdd/repeat.test.js

```diff
@@ -20,13 +20,31 @@ test("should repeat the string count times", () => {
 // Given a target string str and a count equal to 1,
 // When the repeat function is called with these inputs,
 // Then it should return the original str without repetition, ensuring that a count of 1 results in no repetition.
+test("should return the original input with no repetition", () => {
```

Comment:
These unit tests are good, great job on writing them! Now I would like you to think of more test cases, where the values passed to the function are not what you expected. What if you pass integer 3 as `valueToRepeat` or undefined as `numOfTimes`, etc.?